chalicelib/utils.py

```diff
@@ -64,30 +76,28 @@ def standup_workspace(self):
 
     def import_tsv_to_fc(self):
         """
-        Returns a tuple containting the a "participant" dataframe
-        (which needs to be uploaded first), and a dataframe "df",
-        which is technically is the "sample" dataset, and which
-        needs to be uploaded after the participant, to be
-        in FireCloud-compliant format to be uploaded.
-
-        :param url: (str)
-        :param payload: (dict)
-        :param tsv_fname: (str)
-        :param token: (str)
-        :param df: (Pandas dataframe)
-        :return: response objects
+        Uploads "participant", a Pandas Series (which needs
+        to be uploaded first), and a dataframe "sample" (which
+        needs to be uploaded after "participant") to the FC
+        workspace.
         """
         # Check whether the workspace exists in this namespace,
         # otherwise create it.
         if not self.workspace_exists():
```

Comment:
What happens if does exist? Will the import code below still work? It should work if not data has been imported, but what if data has already been imported?